src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -139,7 +139,8 @@ public void CompileMethod(MethodCodeNode methodCodeNodeNeedingCode, MethodIL met
                 {
                     MethodDebugInformation debugInfo = _compilation.GetDebugInfo(methodIL);
 
-                    if (!_compilation.Options.NoLineNumbers)
+                    // TODO: NoLineNumbers
+                    //if (!_compilation.Options.NoLineNumbers)
```

Comment:
Note: I can pipe this through, but do we actually have a use case for this? I think it will be more valuable to have the general "do not generate debug info" switch (that just passes the flag to RyuJIT) instead of trying to restrict what RyuJIT sees.